kratos/modeler/modeler.h

```diff
@@ -68,85 +48,53 @@ class Modeler
     typedef std::size_t SizeType;
     typedef std::size_t IndexType;
 
-    //typedef ModelPart::GeometricalDataContainerType GeometricalDataContainerType;
-
-    //typedef GeometricalDataContainerType::GeometricalDataType GeometricalDataType;
-
-    //typedef ModelPart::GeometryType GeometryType;
-
-    //typedef ModelPart::GeometriesContainerType GeometriesContainerType;
-
-
     ///@}
     ///@name Life Cycle
     ///@{
 
     /// Default constructor.
-    Modeler() {}
+    Modeler(
+        const Parameters ModelerParameters = Parameters())
```

Comment:
```suggestion
        Parameters ModelerParameters = Parameters())
```

that cannot be const either